lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -425,6 +420,15 @@ class JsReactRedux {
   external static ReactContext get ReactReduxContext;
 }
 
+@Props(keyNamespace: '')
+mixin ReduxProviderPropsMixin on UiProps {
+  /// The __single__ Redux store in your application.
+  Store? store;
```

Comment:
Is this actually nullable?

Reply:
Same answer as https://github.com/Workiva/over_react/pull/856#discussion_r1394837924